executor/src/subscriber.rs

```diff
@@ -64,6 +64,7 @@ impl<PublicKey: VerifyingKey> Subscriber<PublicKey> {
             }
             .run()
             .await
+            .unwrap();
```

Comment:
If we are going to have unwraps on failure to run components, we might as well have them be `.unwrap_or_else(..)` with log a message that includes the error?